x-pack/plugins/search_indices/public/components/indices/details_page.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiPageSection, EuiSpacer, EuiButton, EuiPageTemplate } from '@elastic/eui';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useIndex } from '../../hooks/api/use_index';
+
+export const SearchIndexDetailsPage = () => {
+  const indexName = decodeURIComponent(useParams<{ indexName: string }>().indexName);
+
+  const { data: index } = useIndex(indexName);
+
+  return (
+    <EuiPageTemplate
+      offset={0}
+      restrictWidth={false}
+      data-test-subj="searchIndicesDetailsPage"
+      grow={false}
+      bottomBorder={false}
+    >
+      <EuiPageSection>
+        <EuiButton
+          data-test-subj="searchIndexDetailsBackToIndicesButton"
+          color="text"
+          iconType="arrowLeft"
+          onClick={() => {}}
```

Comment:
Do we need a handler for this? this should be `application.navigateToApp('management', { deepLinkId: 'index_management' });` you could add that to a `useCallback()`